app/helpers/applications_helper.rb

```diff
@@ -24,6 +24,12 @@ def scraped_and_received_text(application)
     text
   end
 
+  def decode(description)
```

Comment:
We already do a lot of manipulation to the description in `Application#description`. Since this would put this logic in one place I think we should probably put this there too.